plugins/inputs/lotus/lotus.go

```diff
@@ -383,6 +396,37 @@ func recordTipsetMessagesPoints(ctx context.Context, api api.FullNode, acc teleg
 	return nil
 }
 
+func recordLotusInfoPoints(ctx context.Context, api api.FullNode, acc telegraf.Accumulator, receivedAt time.Time, receivedFrom peer.ID) error {
+	version, err := api.Version(ctx)
+	if err != nil {
+		return err
+	}
+
+	network, err := api.StateNetworkName(ctx)
+	if err != nil {
+		return err
+	}
+	versionTokens := strings.Split(version.Version, "+")
+	if len(versionTokens) != 2 {
+		return fmt.Errorf("developer error, version string has changed format")
```

Comment:
Let's not return an error here. If the version doesn't split, let's fill the version column with everything and leave commit blank.